Hn 17 setup microservice

DaanTul opened the pull request:
Before submitting your pull request, consider the following: Does your code...

- [x] Implement functionality
- [x] Contain comments/documentation
- [x] Build without errors
- [x] Contain and pass unit testing
- [x] Conform to the Style Guide
- [x] Have no merge conflicts with ```master```

When you've checked all of the boxes above your code is up for review. If approved it can be merged with the ```master``` branch.

If you want feedback on your work before having finished your code, add ```[WIP]``` to the title of your PR.

SvenDub commented on crawler/src/main/java/nl/fhict/happynews/crawler/CrawlerController.java:
```diff
@@ -0,0 +1,97 @@
+
+            if(!postRepository.exists(Example.of(p, matcher))) {
+                logger.info("Inserting " + p.getUrl());
+                postRepository.save(p);
```
Should we catch exceptions like `DuplicateKeyException` here, just in case?

DaanTul commented:
Added the duplicate key exception for when it might throw. Maybe it was bug because I changed the url to be unique in the database and some old data was still present.
